src/Triggers/StrawberryJam/OshiroAttackTimeTrigger.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using Microsoft.Xna.Framework;
+using Celeste.Mod.Entities;
+using MonoMod.Cil;
+using MonoMod.RuntimeDetour;
+using MonoMod.Utils;
+using System.Reflection;
+
+namespace Celeste.Mod.CommunalHelper.Triggers.StrawberryJam {
+	[CustomEntity("CommunalHelper/SJ/OshiroAttackTimeTrigger")]
+	public class OshiroAttackTimeTrigger : Trigger {
+		private bool Enable = true;
+		private static ILHook oshiroHook;
+		private static BindingFlags privateInstance = BindingFlags.NonPublic | BindingFlags.Instance;
+		private static MethodInfo oshiroCoroutineInfo = typeof(AngryOshiro).GetMethod("ChaseCoroutine", privateInstance).GetStateMachineTarget();
+		public OshiroAttackTimeTrigger(EntityData data, Vector2 offset) : base(data, offset) {
+			Enable = data.Bool("Enable", true);
+		}
+		public static void Load() {
+			oshiroHook = new ILHook(oshiroCoroutineInfo, ModAttackTime);
+		}
+		public static void Unload() {
+			oshiroHook?.Dispose();
```

Comment:
```suggestion
			oshiroHook?.Dispose();
			oshiroHook = null;
```
Tiny nit, it's better to null the ILHook referance after we are done with it.